error.go

```diff
@@ -21,20 +21,98 @@ type ErrorCode string
+	ErrorCodeInvoiceUpcomingNone                    ErrorCode = "invoice_upcoming_none"
+	ErrorCodeLivemodeMismatch                       ErrorCode = "livemode_mismatch"
+	ErrorCodeMissing                                ErrorCode = "missing"
+	ErrorCodeNotAllowedOnStandardAccount            ErrorCode = "not_allowed_on_standard_account"
+	ErrorCodeOrderCreationFailed                    ErrorCode = "order_creation_failed"
+	ErrorCodeOrderRequiredSettings                  ErrorCode = "order_required_settings"
+	ErrorCodeOrderStatusInvalid                     ErrorCode = "order_status_invalid"
+	ErrorCodeOrderUpstreamTimeout                   ErrorCode = "order_upstream_timeout"
+	ErrorCodeOutOfInventory                         ErrorCode = "out_of_inventory"
+	ErrorCodeParameterInvalidEmpty                  ErrorCode = "parameter_invalid_empty"
+	ErrorCodeParameterInvalidInteger                ErrorCode = "parameter_invalid_integer"
+	ErrorCodeParameterInvalidStringBlank            ErrorCode = "parameter_invalid_string_blank"
+	ErrorCodeParameterInvalidStringEmpty            ErrorCode = "parameter_invalid_string_empty"
+	ErrorCodeParameterMissing                       ErrorCode = "parameter_missing"
+	ErrorCodeParameterUnknown                       ErrorCode = "parameter_unknown"
+	ErrorCodeParametersExclusive                    ErrorCode = "parameters_exclusive"
+	ErrorCodePaymentIntentAuthenticationFailure     ErrorCode = "payment_intent_authentication_failure"
+	ErrorCodePaymentIntentIncompatiblePaymentMethod ErrorCode = "payment_intent_incompatible_payment_method"
+	ErrorCodePaymentIntentInvalidParameter          ErrorCode = "payment_intent_invalid_parameter"
+	ErrorCodePaymentIntentPaymentAttemptFailed      ErrorCode = "payment_intent_payment_attempt_failed"
+	ErrorCodePaymentIntentUnexpectedState           ErrorCode = "payment_intent_unexpected_state"
+	ErrorCodePaymentMethodUnactivated               ErrorCode = "payment_method_unactivated"
+	ErrorCodePaymentMethodUnexpectedState           ErrorCode = "payment_method_unexpected_state"
+	ErrorCodePayoutsNotAllowed                      ErrorCode = "payouts_not_allowed"
+	ErrorCodePlatformAPIKeyExpired                  ErrorCode = "platform_api_key_expired"
+	ErrorCodePostalCodeInvalid                      ErrorCode = "postal_code_invalid"
+	ErrorCodeProcessingError                        ErrorCode = "processing_error"
+	ErrorCodeProductInactive                        ErrorCode = "product_inactive"
+	ErrorCodeRateLimit                              ErrorCode = "rate_limit"
+	ErrorCodeResourceAlreadyExists                  ErrorCode = "resource_already_exists"
+	ErrorCodeResourceMissing                        ErrorCode = "resource_missing"
+	ErrorCodeRoutingNumberInvalid                   ErrorCode = "routing_number_invalid"
+	ErrorCodeSecretKeyRequired                      ErrorCode = "secret_key_required"
+	ErrorCodeSepaUnsupportedAccount                 ErrorCode = "sepa_unsupported_account"
```

Comment:
Last one Remi: I think we want to go with "SEPA" instead of "Sepa" here.

Reply:
We use `Sepa` everywhere else in the library though for SepaDebit. I don't think we should change that one but happy to make the change if you prefer on that one

Reply:
Ah, k. I looked around online and I think "SEPA" is definitely the overall preferred casing (and use in Stripe's docs, etc. too).

That said, given we have pre-existing `Sepa` in stripe-go, I'm good just keeping it like this for consistency.

Reply:
Yeah, I'm unsure on that one, I think both would make sense so deferring to you but happy to make the change

Reply:
K cool. Easiest to go with "Sepa" for now then, so just leave it. Thanks!